Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentMode.cpp

```diff
@@ -542,7 +504,7 @@ namespace WhiteBox
             settingsRegistry->Get(hasTransformMode, WhiteBoxTransformFeature);
             if (hasTransformMode)
             {
-                m_transformModeButtonId = RegisterClusterButton(m_modeSelectionClusterId, "TransformMode");
+                m_transformModeButtonId = RegisterClusterButton(m_modeSelectionClusterId, "Align_to_Object");
```

Comment:
question: Why have we updated this from `TransformMode` to `"Align_to_Object", the previous name seemed better and more accurate

Reply:
might of copied it from somewhere

Reply:
its a temporary icon

Reply:
![image](https://user-images.githubusercontent.com/854359/182008788-1ae17e66-0628-4f8c-b3fa-d6f02574afd9.png)

Reply:
Aaah I see! Sorry about that, thanks for clarifying 👍 (might be worth asking @amzn-leenguy for a new one based on the functionality)

Reply:
Could you leave a comment to explain this just temporarily as that would be useful to understand why it's there, thanks!